### DISCUSSION TITLE
Does coil supports loading app icons?

### DISCUSSION THREAD

**Discussion by ytheekshana** [coil-kt/coil#1373] [Q&A]
I have a recyclerview with an installed apps list. Can we use coil to load the app icons faster & memory efficient way. ?

**Comment by colinrtwhite** [comment]
Not by default, though this could be implemented using a custom `Fetcher`.

**Reply by ytheekshana** [reply]
> Not by default, though this could be implemented using a custom `Fetcher`.

Thank you. Can you give me an example code.
